lapack/gonum/ilaenv.go

```diff
@@ -165,12 +165,12 @@ func (impl Implementation) Ilaenv(ispec int, name string, opts string, n1, n2, n
 				panic(badName)
 			case "TRF":
 				if sname {
-					if n4 <= 64 {
+					if n2 <= 64 {
```

Comment:
How did you find this?

Reply:
Because the test was passing on the first try. Impossible, I thought. Then I found out that the blocked path was not taken because `Ilaenv` was always returning block size of 1 no matter how large the matrix/bandwidth was.